src/main/resources/data-repository-openapi.yaml

```diff
@@ -380,15 +489,15 @@ definitions:
       - follow
     properties:
       name:
-        $ref: '#/definitions/IdentifierNameModel'
+        $ref: '#/definitions/ObjectName'
       tables:
         type: array
         items:
           $ref: '#/definitions/AssetTableModel'
       follow:
         type: array
         items:
-          $ref: '#/definitions/IdentifierNameModel'
+          $ref: '#/definitions/ObjectName'
```

Comment:
add a description (object name type is too vague) - names of the RelationshipModel objects that apply to the assets.
comment about description above: i think we can have multiple paths to the same table from the root object.

comment for DataModelModel name below... can we please call this something else? maybe StudySpecificationModel?

Reply:
I do not think we should allow more than one path to the same table from the root. Our walk of the hierarchy should be predictable. Multiple paths allows for a graph where the walk might not terminate. Better to take the graph case off the table.

Reply:
After discussing, I think we'd like to simplify and tackle the one-path-no-cycle case first.

Reply:
Description updated for the ObjectName definition. Switched to StudySpecificationModel.